src/core/js/models/itemsComponentModel.js

```diff
@@ -66,6 +66,8 @@ define([
         },
 
         setActiveItem: function(index) {
+            if (index === null || index === undefined || index < 0 || index >= this.get('_children').length) return;
```

Comment:
Could just attempt to get the item (needed to toggle it anyway):
```js
var item = this.getItem(index);
if (!item) return;
...
item.toggleActive(true);
```

Reply:
derp yes that would be easier wouldn't it